includes/payment-methods/class-afterpay-payment-method.php

```diff
@@ -24,7 +24,7 @@ class Afterpay_Payment_Method extends UPE_Payment_Method {
 	public function __construct( $token_service ) {
 		parent::__construct( $token_service );
 		$this->stripe_id   = self::PAYMENT_METHOD_STRIPE_ID;
-		$this->title       = __( 'Afterpay', 'woocommerce-payments' );
+		$this->title       = __( 'Afterpay / Clearpay', 'woocommerce-payments' );
```

Comment:
Well, the branding is country dependent. Do we have access to the merchant country here?

Reply:
Discussed here 
https://github.com/Automattic/woocommerce-payments/pull/6398#discussion_r1213259045

Reply:
I have created an https://github.com/Automattic/woocommerce-payments/issues/6425 for the Afterpay / Clearpay switching.